manual/de/ch_Tools_Assistants.xml

```diff
@@ -5082,46 +5082,59 @@
     <title>Kurs-Datenbank</title>
 
     <abstract>
-      <para>Die Kurs-Datenbank listet die Preise eines Handelsguts zu verschiedenen Zeitpunken und ggf. auf
-        verschiedenen Märkten in Bezug zu einem anderen Handelsgut auf. Neue
-        Preise können hinzugefügt oder gelöscht, bestehende Preise bearbeitet und aus
-        einer Auswahl von verschiedenen Quellen im World-Wide-Web abgerufen werden, wenn das
+      <para>Die Kurs-Datenbank listet die Kurse Ihrer Handelsgüter zu verschiedenen Zeitpunkten und ggf. auf
+        verschiedenen Märkten in Bezug zu Ihrer Buchwährung auf. Neue
+        Kurse können hinzugefügt oder gelöscht, bestehende Kurse bearbeitet und aus
+        einer Auswahl von verschiedenen Quellen im Internet abgerufen werden, wenn das
         Handelsgut hierfür richtig konfiguriert ist.
       </para>
+      
+      <para>Auch die Kurse, die Sie mittels &mc.fi.imp.pri-csv; nach &app; importiert haben,
+        werden hier aufgeführt.
+      </para>
     </abstract>
 
-    <para>Mit dem Menüpunkt &mc.to.price; haben Sie die Möglichkeit die Kurse für Ihre Devisen, Aktien
-      und Investmentfonds zu verwalten.
-    </para>
+    <formalpara>
+      <title>Aufruf des Assistenten</title>
+        <para>Mit dem Menüpunkt &mc.to.price; wird er in einem separaten Fenster gestartet.
+        </para>
+    </formalpara>
 
     <glosslist>
+      <title>Begriffe</title>
       <glossentry>
         <glossterm>Handelsgut</glossterm>
         <glossdef>
-          <para>Steht in &app; für alles, was zählt: Währungen, Wertpapiere (d.h. immaterielle Werte wie
+          <para>Steht in &app; für alles, was zählt: Währungen, Wertpapiere (d.h. Werte wie
             Aktien oder Investmentfondsanteile und Anleihen), oder Sachwerte wie Lastwagen und Computer.
           </para>
         </glossdef>
       </glossentry>
 
-      <glossentry>
-        <glossterm>Wechselkurs</glossterm>
+      <glossentry id="tool-price-ge-quote">
+        <glossterm>Kurs</glossterm>
         <glossdef>
-          <para>Wird für den Vergleich zwischen zwei Währungen verwendet.
+          <para>Wird für den Vergleich zwischen zwei Handelsgütern verwendet.
           </para>
         </glossdef>
       </glossentry>
 
       <glossentry>
         <glossterm>Preis</glossterm>
         <glossdef>
-          <para>Kommt zwischen einer Nicht-Währung und einer Währung zur Anwendung.
+          <para>In einem Kauf oder Verkauf realisierter Kurs, also in einer Buchung verwendet wird.
```

Comment:
`, wird also in einer Buchung verwendet.`  oder
`, welcher also in einer Buchung verwendet wird.`